src/corona_hakab_model/manager.py

```diff
@@ -158,37 +166,116 @@ def step(self):
 
         self.simulation_progression.snapshot(self)
 
-    def progress_tests_and_isolation(self, new_tests: List[PendingTestResult]):
+    def progress_isolations(self):
```

Comment:
Don't you think that at least some of this logic should be in the HealthCareManager?